controllers/commonservice_controller.go

```diff
@@ -89,6 +89,9 @@ func (r *CommonServiceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 			if err := r.handleDelete(); err != nil {
 				return ctrl.Result{}, err
 			}
+			if err := r.Bootstrap.DeleteCrossplaneCloudSubscription(r.Bootstrap.CSData.MasterNs); err != nil {
```

Comment:
Is it possible that we have more than one CommonService CRs which configure the `bedrockshim` enabled?

In this case, if we delete one of the CommonService CR, and the Crossplane and Cloud operator will be deleted anyway even there are still CommonService CR which has `bedrockshim` enabled remaining in the cluster.

Reply:
We may follow the same strategy in `r.handleDelete()` and `r.getMinimalSizes()` to decide whether we should delete those operators.
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/operandconfig.go#L363
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/operandconfig.go#L317

Reply:
yea.. I was thinking about this issue, which design they are actually looking for? or they will only have `bedrockshim` in one ns, for example IAF ns.

Reply:
We should consider the multiple adopters scenario. Because IAF will be deployed into each cloudpak namespace. We need to make sure there is no CommonService CR within the scope that enables the crossplane before uninstalling them.